helm/values.yaml

```diff
@@ -0,0 +1,142 @@
+    repository: 506463145083.dkr.ecr.us-west-2.amazonaws.com/cwagent-operator-release
+    tag: latest
+  autoInstrumentationImage:
+    java:
+      # This repository is for internal beta testing only, it will be replaced with the public repository before public release
+      repository: 160148376629.dkr.ecr.us-west-2.amazonaws.com/aws-apm-preview
+      tag: latest
+  ports:
+    containerPort: 9443
+    metricsPort: 8080
+    webhookPort: 9443
+    healthzPort: 8081
+  resources:
+    requests:
+      cpu: 100m
+      memory: 64Mi
+  ## Adds additional environment variables
+  ## e.g ENV_VAR: env_value
+  env:
+    ENABLE_WEBHOOKS: "true"
+
+  # -- Create the manager ServiceAccount
+  serviceAccount:
+    create: true
+    annotations: { }
+
+  podAnnotations: { }
+  podLabels: { }
+
+  service:
+    name:
+
+## Admission webhooks make sure only requests with correctly formatted rules will get into the Operator.
+admissionWebhooks:
+  create: true
+  failurePolicy: Ignore
+  secretName: ""
+
+  ## Defines the sidecar injection logic in Pods.
+  ## - Ignore, the injection is fail-open. The pod will be created, but the sidecar won't be injected.
+  ## - Fail, the injection is fail-close. If the webhook pod is not ready, pods cannot be created.
+  pods:
+    failurePolicy: Ignore
+
+  ## Adds a prefix to the mutating webhook name.
+  ## This can be used to order this mutating webhook with all your cluster's mutating webhooks.
+  namePrefix: ""
+
+  ## Customize webhook timeout duration
+  timeoutSeconds: 10
+
+  ## Provide selectors for your objects
+  namespaceSelector: { }
+  objectSelector: { }
+
+  ## TLS Certificate Option 1: Use Helm to automatically generate self-signed certificate.
+  ## autoGenerateCert must be enabled. This is the default option.
+  ## If true, Helm will automatically create a self-signed cert and secret for you.
+  autoGenerateCert:
+    enabled: true
+    expiryDays: 3650 # 10 years
+
+  ## TLS Certificate Option 2: Use certManager to generate self-signed certificate.
+  ## certManager must be enabled. If enabled, it takes precedence over option 1.
+  certManager:
+    enabled: false
+    ## Provide the issuer kind and name to do the cert auth job.
+    ## By default, OpenTelemetry Operator will use self-signer issuer.
+    issuerRef: { }
+    # kind:
+    # name:
+    ## Annotations for the cert and issuer if cert-manager is enabled.
+    certificateAnnotations: { }
+    issuerAnnotations: { }
+
+  ## Secret annotations
+  secretAnnotations: { }
+  ## Secret labels
+  secretLabels: { }
+
+agent:
+  name:
+  image:
+    repository: public.ecr.aws/cloudwatch-agent/cloudwatch-agent
+    tag: 1.300028.1b210
+  enabled: true
+  serviceAccount:
+    name: # override agent service account name 
+  config: # optional config that can be provided to override the defaultConfig
+  defaultConfig:
+    {
+      "agent": {
+        "debug": true
```

Comment:
we should remove this

Reply:
Yeah I was going to remove these before the final add-on release on Wed